src/hostapi/pulseaudio/pa_linux_pulseaudio.c

```diff
@@ -1262,6 +1275,16 @@ PaError OpenStream( struct PaUtilHostApiRepresentation *hostApi,
 
     if( stream )
     {
+        if ( stream->sourceStreamName )
+        {
+            PaUtil_FreeMemory( stream->sourceStreamName );
```

Comment:
PaUtil_FreeMemory() has a null check internally.

Reply:
Verified and removed null check since it's already being done.